admin-support-cli/tests/check-auto-demotion-action.test.js

```diff
@@ -104,12 +104,13 @@ describe('Check auto demotion', () => {
       replyGithubGetResponse('/repos/test/test/issues/1', null, issueOpenedMock)
       replyGithubPatchResponse('/repos/test/test/issues/1', (_, req) => {
         expect(req.state).toBe('closed')
+        expect.assertions(1)
         return { ...issueOpenedMock, status: 'closed' }
       })
       const action = new CheckAutoDemotionAction(apis, mockParams)
       const result = await action.execute()
       expect(result.status).toBe('success')
-      expect.assertions(2)
+      expect.assertions(1)
```

Comment:
@stebje I don't know if I did this correctly, but the number of expected assertions seemed to fail when the first one was in a closure above? I wonder if something changed with scoping. I reduced the number expected here and then added a new expectation in the block above, but let me know what you think.